helix-core/src/test/java/org/apache/helix/integration/rebalancer/WagedRebalancer/TestWagedRebalance.java

```diff
@@ -354,12 +354,12 @@ public void testMaxPartitionLimitation() throws Exception {
       String limitedResourceName = null;
       int i = 0;
       for (String stateModel : _testModels) {
-        String db = "Test-DB-" + i++;
+        String db = "Test-DB-testMaxPartitionLimitation-" + i++;
```

Comment:
Nit: use getMethodName?

Reply:
Good catch, it was added while I didn't notice there is getMethodName.